apps/site/pages/components/organisms/product-grid.mdx

```diff
@@ -0,0 +1,203 @@
+  const components = ['ProductGrid.tsx', 'ProductGridItem.tsx']
+  const [productGridProps, productGridItemProps] = getComponentPropsFrom(
+    productGridPath,
+    components
+  )
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        productGridProps,
+        productGridItemProps,
+      },
+    },
+  }
+}
+
+export const ProductGridPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { productGridProps, productGridItemProps } = useSSG()
+  return {
+    ProductGrid: <PropsSection propsList={productGridProps} />,
+    ProductGridItem: <PropsSection propsList={productGridItemProps} />,
+  }[component]
+}
+
+<header>
+
+# Product Grid
+
+It's a section generally used on PLP pages to list the products available in the store.
+
+</header>
+
+The `ProductGrid` is a compound of the following:
+
+- `ProductGrid`: wraps the product grid items.
+- `ProductGridItem`: wraps the product inside a list item.
+
+\*We recommend using the [ProductCard](/components/molecules/product-card) as the direct child of the ProductGridItem.
+
+---
+
+## Import
+
+Import the component from [@faststore/ui](/../components)
+
+```tsx
+import { ProductGrid, ProductGridItem } from '@faststore/ui'
+```
+
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/organisms/ProductGrid/styles.scss'
+```
+
+---
+
+## Usage
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection>
+      <ProductGrid>
+           {Array.from({ length: 5 }, (_, index) => (
+             <ProductGridItem>
+               <ProductCard>
+                 <ProductCardImage>
+                   <img
+                     data-fs-image
+                     src={product.image[0].url}
+                     alt={product.image[0].alternateName}
+                   />
+                 </ProductCardImage>
+                 <ProductCardContent
+                   title={product.isVariantOf.name}
+                   price={{
+                     value: product.offers.offers[0].price,
+                     listPrice: product.offers.offers[0].listPrice,
+                     formatter: useFormattedPrice,
+                   }}
+                 />
+               </ProductCard>
+             </ProductGridItem>
+           ))}
+       </ProductGrid>
+
+    </OverviewSection>
+
+  </Tab>
+  <Tab>
+
+    ```tsx
+
+<ProductGridItem>
+  <ProductCard>
+    <ProductCardImage>
+      <img
+        data-fs-image
+        src={product.image[0].url}
+        alt={product.image[0].alternateName}
+      />
+    </ProductCardImage>
+    <ProductCardContent
+      title={product.isVariantOf.name}
+      price={{
+        value: product.offers.offers[0].price,
+        listPrice: product.offers.offers[0].listPrice,
+        formatter: useFormattedPrice,
+      }}
+    />
+  </ProductCard>
+</ProductGridItem>
+```
```

Comment:
Fan, this indentation looks odd on the doc, could you check?
<img width="870" alt="image" src="https://user-images.githubusercontent.com/11613011/232615236-9fdfac0f-29ae-43c8-8978-5e137d5ddf4a.png">

Reply:
omg, that's true! I'll fix it!